src/Sdk/DTWebApi/WebApi/TaskAgentHttpClient.cs

```diff
@@ -64,6 +64,7 @@ public class TaskAgentHttpClient : TaskAgentHttpClientBase
             Guid lockToken,
             DateTime finishTime,
             TaskResult result,
+            Guid targetHostId,
```

Comment:
we need to figure out a different way of doing this, can punt til next quarter maybe